api/apps/geoprocessing/src/modules/tile/tile.service.ts

```diff
@@ -131,8 +134,8 @@ export class TileService {
     if (result) {
       return result;
     } else {
-      this.logger.debug(query.getSql());
-      throw new Error("Property 'mvt' does not exist in res.rows[0]");
+      this.logger.error(query.getSql());
+      throw new NotFoundException("Property 'mvt' does not exist in res.rows[0]");
```

Comment:
Throwing HttpExceptions inside services suggests it can be used only in the HTTP context, which is not always the case.

I believe it should be kept for the controller. But, as far as I saw, it's common in the project to do like this.